[rankmyapp @ 3.9.0] 06/06/18 18:22:44 /getup-engine $ diff -pu /openshift-ansible/playbooks/openshift-hosted/private/openshift_default_storage_class.yml /openshift-ansible/playbooks/openshift-hosted/private/openshift_default_storage_class.yml.new
--- /openshift-ansible/playbooks/openshift-hosted/private/openshift_default_storage_class.yml	2018-06-06 18:22:04.708005065 +0000
+++ /openshift-ansible/playbooks/openshift-hosted/private/openshift_default_storage_class.yml.new	2018-06-06 18:21:54.100986148 +0000
@@ -5,4 +5,4 @@
   - role: openshift_default_storage_class
     when:
     - openshift_cloudprovider_kind is defined
-    - openshift_cloudprovider_kind in ['aws','gce','openstack','vsphere']
+    - openshift_cloudprovider_kind in ['aws','azure','gce','openstack','vsphere']
